package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "s47_maybe_i_know_my_place",
+  "version": "1.0.0",
+  "description": "Topic: Maybe I Know my place",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
```

Comment:
Consider specifying the "author" field in your package.json. This can be useful for others who may want to contribute to your project in the future. [medium]